lean/components/docker/docker_manager.py

```diff
@@ -51,16 +51,24 @@ def pull_image(self, image: DockerImage) -> None:
             return
 
         self._logger.info(f"Pulling {image}...")
+        
+        import platform
+        isArchitectureARM = True if platform.machine().startswith('arm') else False
+        isForceAmdRequired = isArchitectureARM and self._platform_manager.is_system_macos()
         # We cannot really use docker_client.images.pull() here as it doesn't let us log the progress
         # Downloading multiple gigabytes without showing progress does not provide good developer experience
         # Since the pull command is the same on Windows, macOS and Linux we can safely use a system call
         if which("docker") is not None:
-            process = run(["docker", "image", "pull", str(image)])
+            args = ["docker", "image", "pull", str(image)]
+            if isForceAmdRequired:
```

Comment:
I think safe to do this regardless right?  We need this version for all chipsets

Reply:
Right 💯 ty